llvm/include/llvm/MC/MCSection.h

```diff
@@ -107,6 +115,11 @@ class MCSection {
   };
   SmallVector<PendingLabel, 2> PendingLabels;
 
+  // Mapping from subsection number to fragment list. At layout time, the
+  // subsection 0 list is replaced with concatenated fragments from all
+  // subsections.
+  SmallVector<std::pair<unsigned, std::unique_ptr<FragList>>, 1> Subsections;
```

Comment:
Why a unique_ptr? Whenever Subsections is updated, also CurFragList is updated, so no dangling pointer could occur?

Reply:
Thx. Switched to plain FragList.